README.rst

```diff
@@ -2695,34 +2695,34 @@ since only filtered data is sent to the requester.
 
 **Predicate Module Operators**
 
-The ``Predicate`` module offered by the Python client includes many
+The ``predicate`` module offered by the Python client includes many
 operators for your query requirements. Some of them are explained below.
 
-- ``is_equal_to``: Checks if the result of an expression is equal to a
+- ``equal``: Checks if the result of an expression is equal to a
   given value.
-- ``is_not_equal_to``: Checks if the result of an expression is not
+- ``not_equal``: Checks if the result of an expression is not
   equal to a given value.
-- ``is_instance_of``: Checks if the result of an expression has a
+- ``instance_of``: Checks if the result of an expression has a
   certain type.
-- ``is_like``: Checks if the result of an expression matches some
+- ``like``: Checks if the result of an expression matches some
   string pattern. ``%`` (percentage sign) is the placeholder for many
   characters, ``_`` (underscore) is placeholder for only one character.
-- ``is_ilike``: Checks if the result of an expression matches some
+- ``ilike``: Checks if the result of an expression matches some
   string pattern in a case-insensitive manner.
-- ``is_greater_than``: Checks if the result of an expression is greater
+- ``greater_than``: Checks if the result of an expression is greater
   than a certain value.
-- ``is_greater_than_or_equal_to``: Checks if the result of an
+- ``greater_equal``: Checks if the result of an
   expression is greater than or equal to a certain value.
-- ``is_less_than``: Checks if the result of an expression is less than
+- ``less_than``: Checks if the result of an expression is less than
   a certain value.
-- ``is_less_than_or_equal_to``: Checks if the result of an expression
+- ``less_equal``: Checks if the result of an expression
   is less than or equal to a certain value.
-- ``is_between``: Checks if the result of an expression is between two
+- ``between``: Checks if the result of an expression is between two
   values (this is inclusive).
-- ``is_in``: Checks if the result of an expression is an element of a
+- ``in_``: Checks if the result of an expression is an element of a
```

Comment:
Do you mean something like this?
```python
class _Predicates:
    def and(self, ...):
        pass

Predicates = _Predicates()  # import and use this
```
This is not a valid Python code. I might also misunderstand your suggestion. Could you give a sample code?

For the other question, yes it is pretty common practice to that. For example, take a look at the following link, https://docs.sqlalchemy.org/en/13/core/sqlelement.html, it is a pretty popular Python package

Reply:
Yes, I mean something similar to what we have in Node.js client: https://github.com/hazelcast/hazelcast-nodejs-client/blob/master/code_samples/paging_predicate.js#L75

Not sure if that's a good idea to do the same in Python.

Reply:
You can already use this version like that
```python
from hazelcast import predicate

p = predicate.equal("a", 2)
```

But, I believe imported functions are easier to use(require less typing and more readable IMHO)

Reply:
With TypeScript, it's more convenient to use a single object. But in case of Python it may be another story, so I get the point of keeping things as is.